go.sum

```diff
@@ -1,4 +1,6 @@
 github.com/MOHC-LTD/shopify v1.18.0 h1:T9Gfa8i5ibInjgTvjr/EUIPqcyPSxYxcBanFbLVtYkk=
 github.com/MOHC-LTD/shopify v1.18.0/go.mod h1:ZqEEOCrFlhlQxqxzpfE9lah5uvjiGQNvcGERxaTACPQ=
+github.com/MOHC-LTD/shopify v1.18.1 h1:Wm6d1OIzObNOxR++UtfKz2dsGTN6cTcHb2zjR31Z7ho=
+github.com/MOHC-LTD/shopify v1.18.1/go.mod h1:ZqEEOCrFlhlQxqxzpfE9lah5uvjiGQNvcGERxaTACPQ=
```

Comment:
Do you need to run `go mod tidy`? Looks like you have duplicates here?

Reply:
I did indeed need to